### PR TITLE
Fix/issue 2518

### PR DESCRIPTION
**Pull request by rapemer**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ bin/*
 !bin/install
 !bin/sylius-cs-fixer
 vendor
+/.idea/
```
Comment (pjedrzejewski) [inline .gitignore]: We do not add IDE directories to gitignore (it would become too messy if everyone adds his IDE!), could you add it to .git/info/exclude on your local repository and revert this change? Thank you!

Comment (amenophis) [inline .gitignore]: You can also use a global ignore file.
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

File: src/Sylius/Bundle/SearchBundle/Finder/OrmFinder.php [hunk 2]
```diff
@@ -268,7 +268,9 @@ public function buildFacet($idsFromOtherFacets, $facets, $givenFacetName, $resul
                             }
                         }
                     }
-                    asort($calculatedFacets[$name]);
+                    if (isset($calculatedFacets[$name]) && !is_null($calculatedFacets[$name])) {
```
Comment (stloyd) [inline src/Sylius/Bundle/SearchBundle/Finder/OrmFinder.php]: This `is_null()` is useless as `isset()` don't allow `null` anyway... http://3v4l.org/5CPoY

**Comment by pjedrzejewski** [comment]
@rapemer Thank you for your contribution, I will be happy to merge it after you apply these few tweaks mentioned in the comments. :)

File: .gitignore (hunk 1 above)
Comment (rapemer) [inline .gitignore]: OK guys, thank you for advice!

File: src/Sylius/Bundle/SearchBundle/Finder/OrmFinder.php (hunk 2 above)
Comment (rapemer) [inline src/Sylius/Bundle/SearchBundle/Finder/OrmFinder.php]: @stloyd: yep, you have right ;-)

**Comment by rapemer** [comment]
@pjedrzejewski: done, you can merge it. Thanks for advices, I will try to help with contributing as I can ;-)

**Comment by pjedrzejewski** [comment]
@rapemer Have you pushed the changes? :)

**Comment by rapemer** [comment]
@pjedrzejewski: damn, I pushed changes to master branch

**Comment by aRn0D** [comment]
ping @rapemer, can you squash your commits please ?

**Comment by aRn0D** [comment]
ping @rapemer

**Comment by pjedrzejewski** [comment]
Ping @rapemer. :+1:
